### ISSUE TITLE
May be the reason for the not good results

### ISSUE THREAD

**Issue by cjplol** [mousecpn/FERNet-Pytorch#6]
Hi, I've tried to train underwater datasets with your code.As you said before,it didn't obtain the good result. I reviewed the original paper and your code, and I think it may be due to the training strategy. Section 4.1 of the paper describes how it trains, but your code uses 'configs/_base_/schedule_2x.py', which I think that's probably the problem.

**Comment by mousecpn** [comment]
> 

Yes, our training strategy is different from the original paper. However, the performance can not defeat SSD, and the model is modified from it. Thus, I don't think that is the main problem.
